packages/restapi/src/lib/user/createUser.ts

```diff
@@ -0,0 +1,60 @@
+import axios from "axios";
+import { generateKeyPair, walletToCAIP10 } from "../chat/helpers";
+import Constants from "../constants";
+import { encryptWithRPCEncryptionPublicKeyReturnRawData, getAPIBaseUrls } from "../helpers";
+import { getPublicKey } from "../helpers";
+import { IUser } from "../types";
+import { get } from "./getUser";
+
```

Comment:
/**
 *  POST /v1/users/
 */
 
 we can add the comment here

Reply:
done